hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpGetCommandProcessor.java

```diff
@@ -78,7 +79,12 @@ private void handleMap(HttpGetCommand command, String uri) {
         int indexEnd = uri.indexOf('/', URI_MAPS.length());
         String mapName = uri.substring(URI_MAPS.length(), indexEnd);
         String key = uri.substring(indexEnd + 1);
-        Object value = textCommandService.get(mapName, key);
+        Object value;
+        if(MAP_ENTRIES_COMMAND.equalsIgnoreCase(key)){
```

Comment:
Hi @sancar 
i think it is better if  we change to 
` /hazelcast/rest/maps-command/mapName/entries|size|keys|clear`

it is more useful for common usage with this solution.

For queues ;
`/hazelcast/rest/queues-command/queueName/size|iterator|clear `

What do you think about it?

if you think the same way , i will open new PR .